.github/workflows/test.yaml

```diff
@@ -0,0 +1,92 @@
+name: Trigger test suite
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+          lfs: 'true'
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.17.6'
+
+      - name: Install protobuf
+        run: |
+          go install github.com/golang/protobuf/protoc-gen-go@v1.3.5
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y protobuf-compiler
+
+      - name: Cache go modules
+        id: cache-go-mod
+        uses: actions/cache@v2.1.5
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Cache ffmpeg
+        id: cache-ffmpeg
+        uses: actions/cache@v2.1.5
+        with:
+          path: ~/compiled
+          key: ${{ runner.os }}-ffmpeg-${{ hashFiles('**/install_ffmpeg.sh') }}
+          restore-keys: |
+            ${{ runner.os }}-ffmpeg
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update \
+            && sudo apt-get install -y software-properties-common curl apt-transport-https \
+            && sudo apt-key adv --keyserver keyserver.ubuntu.com --recv 15CF4D18AF4F7421 \
+            && sudo add-apt-repository "deb [arch=amd64] http://apt.llvm.org/xenial/ llvm-toolchain-xenial-8 main" \
+            && sudo apt-get update \
+            && sudo apt-get -y install clang-8 clang-tools-8 build-essential pkg-config autoconf git python
+
+          sudo update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-8 30 \
+            && sudo update-alternatives --install /usr/bin/clang clang /usr/bin/clang-8 30
+
+      - name: Install go modules
+        run: go mod download
+
+      - name: Install ffmpeg
```

Comment:
Previously we always ran this test workflow after the Linux build step so that we could use cached deps for a faster test workflow. Possible to continue doing this instead of repeating the dep installation process here that was already done for the Linux build step?